go.mod

```diff
@@ -5,18 +5,18 @@ require (
 	github.com/btcsuite/btcd v0.20.1-beta
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
 	github.com/cosmos/go-bip39 v0.0.0-20180819234021-555e2067c45d
-	github.com/go-kit/kit v0.8.0 // indirect
-	github.com/gorilla/websocket v1.4.0
+	github.com/ethereum/go-ethereum v1.9.13
```

Comment:
do we need this dependency

Reply:
I provided an util to convert eth header to bsc header.